tablo/interfaces/arcgis/views.py

```diff
@@ -212,13 +212,15 @@ def handle_request(self, request, **kwargs):
 
 
 class QueryView(FeatureLayerView):
-    query_limit_default = 50000
+    query_limit_default = 1000
 
     def handle_request(self, request, **kwargs):
-
         search_params = {}
-        limit = int(kwargs.get('limit', self.query_limit_default))
-        offset = int(kwargs.get('offset', 0))
+
+        # Capture query bounding parameters (limit/offset does not apply when filtering by object ids)
+        object_ids = kwargs['objectIds'].split(',') if kwargs.get('objectIds') else []
+        limit = 0 if object_ids else int(kwargs.get('limit') or self.query_limit_default)
```

Comment:
This will ignore the limit when requesting features BY object ID, but this will still limit a request FOR object IDs, which shouldn't be limited. This kind of request:

/tablo/arcgis/rest/services/901/FeatureServer/0/query?outFields=*&f=json&outSR=3857&spatialRel=esriSpatialRelIntersects&returnGeometry=true&returnIdsOnly=true&where=1=1

I know it's an odd request with conflicting items, but this is what will be sent by a default query for object IDs if done through the ArcGIS interface.